core/js/typespecs/app_types.ts

```diff
@@ -1219,6 +1239,41 @@ export const createApiKeyOutputSchema = apiKeySchema
   )
   .openapi("CreateApiKeyOutput");
 
+export const createServiceTokenSchema = z.object({
+  name: z
+    .string()
+    .describe("Name of the service token. Does not have to be unique"),
+  org_id: z
```

Comment:
most other APIs like `createApiSchema` ask for an `org_name` instead of an `org_id`. Should we do that here?